src/main/java/picard/vcf/GatherVcfs.java

```diff
@@ -211,7 +221,8 @@ private static void gatherWithBlockCopying(final List<File> vcfs, final File out
 
                 // a) It's good to check that the end of the file is valid and b) we need to know if there's a terminator block and not copy it
                 final BlockCompressedInputStream.FileTermination term = BlockCompressedInputStream.checkTermination(f);
-                if (term == BlockCompressedInputStream.FileTermination.DEFECTIVE) throw new PicardException(f.getAbsolutePath() + " does not have a valid GZIP block at the end of the file.");
+                if (term == BlockCompressedInputStream.FileTermination.DEFECTIVE)
```

Comment:
It's nice to have these formatted across lines but now it's more obvious that they're missing `{` `}`'s